src/theme.js

```diff
@@ -39,11 +39,24 @@ const colorTheme = createTheme({
     },
   },
 });
-
+const theme = createTheme({
+  components: {
+    // Name of the component
+    MuiButton: {
+      styleOverrides: {
+        // Name of the slot
+        root: {
+          // Some CSS
+          letterSpacing: 0,
+        },
+      },
+    },
+  },
+});
```

Comment:
is it possible to declare two const with the same name?

Reply:
No, it is not.